lib/preview-web/src/PreviewWeb.test.ts

```diff
@@ -1022,6 +1022,23 @@ describe('PreviewWeb', () => {
     });
   });
 
+  describe('onPreloadStories', () => {
+    it('loads stories', async () => {
+      document.location.search = '?id=component-one--a&viewMode=docs';
+      const mockedImportFn = jest.fn(() =>
+        Promise.resolve({ default: { title: 'Component Two' }, C: () => {} } as any)
+      );
+      const preview = await createAndRenderPreview({
+        importFn: mockedImportFn,
+      });
+      await preview.onPreloadStories(['component-two--c']);
+
+      await waitForRender();
+
+      expect(mockedImportFn).toHaveBeenCalledWith('./src/ComponentTwo.stories.js');
```

Comment:
This a little confusing as the initial selection `component-one--a` isn't actually going to be returned from the first render, I guess it shows an error but we aren't checking? 

```suggestion
      document.location.search = '?id=component-one--a&viewMode=docs';
      const preview = await createAndRenderPreview();
      await waitForRender();

      importFn.mockClear();
      await preview.onPreloadStories(['component-two--c']);
      expect(importFn).toHaveBeenCalledWith('./src/ComponentTwo.stories.js');
```

I think this is a bit simpler?